README.md

```diff
@@ -1,2 +1,39 @@
 # TcCommando
 A wrapper for using a PLCopen-like interface with sequencers, functions, and more.
+
+## Version Information
+This documentation is for version `0.01.0` of the TcCommando library.
+
+## Overview
+TcCommando provides a way to abstract and monitor an object's commands in your TwinCAT solution. TcCommando decouples function calls and response monitoring from implementation. This enables remote function calls, multiple instances of function calls with separate status monitoring, and ways to group calls together and manage their status as a group. 
```

Comment:
Is it intentional that the README and Overview page are essentially the same? This may make it easy for one page to become out of date/sync with the other.

Reply:
This was the intention at the time but I think you are right that it makes more sense to keep the readme lighter and point to the docs folder for more in-depth info